### ISSUE TITLE
Incorrect biomaterial mapping (cell suspension <-> file)

### ISSUE THREAD

**Issue by pnejad** [ebi-ait/hca-ebi-wrangler-central#513]
**Description of the task:**

@hannes-ucsc  pointed out that these 4 datasets have at least 1 specimen which was the sequencing input. 

https://data.humancellatlas.org/explore/projects/6c040a93-8cf8-4fd5-98de-2297eb07e9f6 **(EBI)**
https://data.humancellatlas.org/explore/projects/71eb5f6d-cee0-4297-b503-b1125909b8c7  **(EBI)**
https://data.humancellatlas.org/explore/projects/c4077b3c-5c98-4d26-a614-246d12c2e5d7  **(EBI)**
https://data.humancellatlas.org/explore/projects/7adede6a-0ab7-45e6-9b67-ffe7466bec1f **(UCSC)**

7adede6a-0ab7-45e6-9b67-ffe7466bec1f - The specimen ID was used accidentally instead of the cell suspension ID in the sequence file tab. @rachadele will update this and resubmit the dataset. 

For the other 3 datasets, I was not able to download the submitted spreadsheets from ingest (_Error - Service Unavailable_) to troubleshoot. I did find other things that need to be fixed based on the info shown on the data portal project pages. We can discuss this further during our next wrangler call.

**Comment by ESapenaVentura** [comment]
@Wkt8 I can confirm the 3 EBI datasets were Bulk + single cell RNA seq!

**Comment by pnejad** [comment]
@hannes-ucsc  cell suspensions are not limited to single cells. So even for bulk experiments, the specimen is processed into suspensions of cells before the bulk-RNA-seq is carried out.

**Comment by willrockout** [comment]
@Wkt8 @ESapenaVentura Why would you set specimen as sequencing input for bulk data? A cell suspension is just a pool of cells that would still need to be created for library prep even in bulk. The only difference is they didn't separate them into single cells but that's how we handle 10x data.

**Comment by pnejad** [comment]
I do not agree with Mallory's blood example. Blood that is collected from a donor still needs to be enriched for PBMC's and put into cell suspensions before the library prep protocol is carried out. 

I do agree with her statement that "...there will most likely never be cell suspensions prior to an imaging assay". But then again, I don't think an imaged specimen would be the sequencing_input. Wranglers - I'm not an expert when it comes to imaging assays, so please correct me if I'm wrong here.

**Comment by hannes-ucsc** [comment]
@ami-day

**Comment by ami-day** [comment]
@ESapenaVentura is going to test this

**Comment by ESapenaVentura** [comment]
@ami-day what? we didn't discuss this on stand-up

**Comment by ofanobilbao** [comment]
@ESapenaVentura is this still required? Do you know?

**Comment by ESapenaVentura** [comment]
This is still needed
